presto-docs/src/main/sphinx/security/password-file.rst

```diff
@@ -27,7 +27,7 @@ The following configuration properties are available:
 ==================================== ==============================================
 Property                             Description
 ==================================== ==============================================
-``file.password``                    Path of the password file.
+``file.password-file``                    Path of the password file.
 
 ``file.refresh-period``              How often to reload the password file.
                                      Defaults to ``1m``.
```

Comment:
Oh. So this is where the default came from. Should we consider changing file based access rules to have this default set as well?

Reply:
Based on feedback from david I changed the docs to 5s, and changed the file group provider to 5s also.